.github/workflows/spark_sql_test.yml

```diff
@@ -79,3 +79,41 @@ jobs:
         env:
           LC_ALL: "C.UTF-8"
 
+  spark-sql-native-iceberg-compat:
+    strategy:
+      matrix:
+        os: [ubuntu-24.04]
+        java-version: [11]
+        spark-version: [{short: '3.5', full: '3.5.5'}]
```

Comment:
This is intentional to just run with one Spark version for now. Once all tests are passing for 3.5.5 we can extend to other versions.